deploy/cloud-init.yaml

```diff
@@ -0,0 +1,47 @@
+#cloud-config
+---
+users:
+- name: egi
+  gecos: EGI
+  primary_group: egi
+  groups: users
+  shell: /bin/bash
+  sudo: ALL=(ALL) NOPASSWD:ALL
+  ssh_import_id:
+  - gh:enolfc
+  - gh:gwarf
+
+apt:
+  sources:
+    ansible-ppa.list:
+      source: "deb http://ppa.launchpad.net/ansible/ansible/ubuntu xenial main"
+      keyid: 6125E2A8C77F2818FB7BD15B93C4A3FD7BB9C367
+
+packages:
+- git
+- ansible
+- jq
+
+write_files:
+- content: |
+    #!/bin/sh
+
```

Comment:
Should we run with things like `set -e`?

Reply:
Yes. Needs a bit of better error handling yet. I will add something